package.json

```diff
@@ -19,6 +19,7 @@
     "http-proxy-middleware": "^2.0.0",
     "interweave": "^12.7.2",
     "jwt-decode": "^3.1.2",
+    "latest": "^0.2.0",
```

Comment:
I think you may have accidentally installed this when you were trying to upgrade npm. We do not want this dependency.